contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -182,6 +186,15 @@ contract VestingRegistryLogic is VestingRegistryStorage {
                 uint256(VestingType.Vesting),
                 _vestingCreationType
             );
+
+        vestingCreationAndTypes[vesting] = VestingCreationAndTypeDetail(
```

Comment:
vestingCreationAndTypes[vesting] = VestingCreationAndTypeDetails(
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)